riff-raff/app/assets/javascripts/notifications.coffee

```diff
@@ -0,0 +1,17 @@
+Notification.requestPermission()
+
+intervalId = null
+
+checkStatus = () ->
+  if $('[data-run-state]').length != 0
+    buildName = window.riffraff.buildName
+    switch $('[data-run-state]').data('run-state')
+      when 'Failed' then new Notification('Riffraff', {body: 'Deployment of ' + buildName + ' failed!'})
+      when 'Completed' then new Notification('Riffraff', {body: 'Deployment of ' + buildName + ' is finished'})
+    disableCheck()
+
+if $('[data-run-state]').length == 0
+  intervalId = setInterval checkStatus, 800
```

Comment:
yes I want it to be out of sync from the main refresh loop (1s) otherwise we can have up to 2s of latency. (I admit it's a detail, but hey why not)